src/Makani.Docs/Program.cs

```diff
@@ -1,16 +1,17 @@
-using Makani;
 using Makani.Docs;
 using Microsoft.AspNetCore.Components.Web;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using Serilog;
 using Serilog.Events;
+using PureBlazor.Components;
 
 var builder = WebAssemblyHostBuilder.CreateDefault(args);
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
 builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
-builder.Services.AddMakani();
+builder.Services.AddPureBlazor();
```

Comment:
Since we may eventually have a Nuget package for the actual CMS, let's name this `AddPureBlazorComponents()`